clang-tools-extra/clang-tidy/llvmlibc/InlineFunctionDeclCheck.cpp

```diff
@@ -79,6 +79,10 @@ void InlineFunctionDeclCheck::check(const MatchFinder::MatchResult &Result) {
     if (MethodDecl->getParent()->isLambda())
       return;
 
+  // Ignore functions that have been deleted.
+  if (FuncDecl->isDeleted())
```

Comment:
Instead of this simply use:
```
void InlineFunctionDeclCheck::registerMatchers(MatchFinder *Finder) {
  Finder->addMatcher(functionDecl(unless(isDeleted())).bind("func_decl"), this);
}
```